samples/Actor/DemoActor/DemoActor.cs

```diff
@@ -100,13 +115,21 @@ protected override Task OnDeactivateAsync()
             return Task.CompletedTask;
         }
 
-        // This method is called when the timer is triggered based on its registration.
-        // It updates the PropertyA value.
-        private Task TimerCallBack(object data)
+        /// <summary>
+        /// This method is called when the timer is triggered based on its registration.
+        /// It updates the PropertyA value.
+        /// </summary>
+        /// <param name="data">Timer input data.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public Task TimerCallback(object data)
```

Comment:
did we talk about changing the signature of this to just be `byte[]`? that would make it really clear what the contract is.